google/ggrc-core
# GGRC-6248 Forward issuetracked objects, created by import to background task

# Dependencies

This PR is `on hold` until the dependencies are merged:

- [x] ~~https://github.com/google/ggrc-core/pull/8586~~

# Issue description

We should forward all issuetracked objects that were imported to our IssueTracker bulk update mechanism. 
# Steps to test the changes

As now it's not possible to turn integration on/off via import:
1. Create issuetracked instanses with enabled Issue Tracker integration. 
2. Import these instanses with some changes. Check if appropriate tickets were updated in Issue Tracker. 
3. Check email notification. It should be sent to your gmail. 
4. To check error notification set invalid component id. 
# Solution description

1. As we should send email notifications, I've added import_export job and user that has started import to import converter. This information would be forwarded into bulk mechanism. 
2. I've collected all objects using revisions as it was mentioned in our tech doc and formatted to make it possible to send it to bulk mechanism.
# Sanity checklist




# PR Review checklist
